.go-version

```diff
@@ -1 +1 @@
-1.16.2
\ No newline at end of file
+1.17.0
```

Comment:
I believe that now Go v1.19 has been released that we can begin to bump the utility providers to Go v1.18.
We should be ok to specify `1.18.5`  👍 

We should also be ok to update `test.yml` to:
```yml
go-version: [ '1.18', '1.19' ]
```